isis/tests/FunctionalTestsTgocassisstitch.cpp

```diff
@@ -37,7 +40,7 @@ TEST(TgoCassisstitch, TgoCassisstitchMultiframeTest) {
     FAIL() << "Unable to run tgocassisstitch with cube list: " << e.what() << std::endl;
   }
 
-  Cube cube(prefix.path() + "/CAS-MCO-2016-11-22T16:16:16.833.cub");
+  Cube cube(prefix.path() + "/CAS-MCO-default.cub");
```

Comment:
Could we only change the result of one of these tests? Basically only set the cubename in one of them rather than both?

Reply:
should be addressed with the latest commit

Reply:
Looks like this got unresolved?